base/data/nlu/tbcheck.md

```diff
@@ -21,3 +21,13 @@
 - D
 - E
 - S1
```

Comment:
@Buhle79 I thought you were going to try TB_S1 as per https://praekeltfoundation.slack.com/archives/C0188F7MUPN/p1675943919885059?thread_ts=1675927778.824159&cid=C0188F7MUPN

Reply:
I've tested locally it's now working, i've missed the nlu models when deploying

Reply:
Ah I see. 👍